app/models/repository.rb

```diff
@@ -61,12 +61,14 @@ def self.add_repo(event, namespace, repo, tag)
     if repository.nil?
       repository = Repository.create(namespace: namespace, name: repo)
     elsif repository.tags.exists?(name: tag)
-      return
+      repository.tags.where(name: tag).each { |tag| tag.touch }
```

Comment:
@Salzig well, not always. Note that when the actor could not be found, it also returns nil. I think that we should return nil when no repo has been added.